completions/ant

```diff
@@ -1,5 +1,30 @@
 # bash completion for ant and phing                        -*- shell-script -*-
 
+_ant_parse_targets()
+{
+    local line basedir
+
+    basedir=$(dirname "$1")
```

Comment:
Looks otherwise ok to me, but please don't use `dirname`. I think `basedir=${1%/*}` should work instead of it.